### DISCUSSION TITLE
Can't get this Python working

### DISCUSSION THREAD

**Discussion by Brom2855** [mu-editor/mu#2390] [Q&A]
Mu app recommended, went and tried its installer but it seems to be bugged: can't install, just offers retry or cancel. The "previous beta versions" on the official website redirect to here and so much irrelevant info. I just want to know how to install Mu, nothing complex.

Windows 10 user, btw.

**Comment by ntoll** [comment]
Hi @Brom2855 - "seems to be bugged". We need more information (like a detailed description of what happens, any error messages, and so on), if we're to be able to fix this.

**Reply by Brom2855** [reply]
Download installer
Run it
Progress bar doesn't fill, within 30 seconds pop-up window says installer - retry or cancel

**Reply by ntoll** [reply]
Is there no other message describing what's happened..? (This isn't a lot to go on) :-/ @tmontes An suggestions for ways to get further information from the installer?

Just to confirm, you're using Windows 10 64bit..? The latest version of Mu..?

**Reply by Brom2855** [reply]
Do u need a screenie or something? Yes, windows 1064bit and latest version of Mu.

**Reply by carlosperate** [reply]
A screenshot of the error message would be very useful, yes, thank you!
